src/components/Flipper/Flipper.tsx

```diff
@@ -264,18 +332,29 @@ function Flipper() {
                                     <span style={{ cursor: "pointer" }}><DeleteIcon color="error" /></span>
                                 </span>
                             </Form.Group>
+                            <p>The flipper is stuck <Tooltip type="hover" content={<HelpIcon />} tooltipContent={<span>We get new auctions every 60sek. from Hypixel. So you may have to wait a bit for a new batch to be found.</span>} /></p>
```

Comment:
this pushes the flips down to far. Either remove all scroll to top (e.g. AuctionDetails) or make sure the copy button is always visible on full HD screens by squashing filter options (putting them in one line with linewarp)